EIPS/eip-7934.md

```diff
@@ -0,0 +1,77 @@
+---
+eip: 7934
+title: RLP Execution Block Size Limit
+description: Introduce a protocol-level cap on the maximum RLP-encoded block size to 10 MB, including a 512 KB margin for beacon block size.
```

Comment:
I think the correct unit to be used in this document is `KiB` and `MiB`. A MegaByte is `1e6` bytes, and a MebiByte is `2^20 = 1,048,576` bytes. See https://simple.wikipedia.org/wiki/Mebibyte (KiB = 1024 bytes, KB = 1000 bytes)